### PR TITLE
#2.15 förbättra vatten2

### PR DESCRIPTION
**Pull request by Hansson00**
Pull request nr2

### CODE REVIEW

**Comment by Manfred-Hastmark** [comment]
Lagspike when going in and out of water with 10 planets

image

**Comment by Hansson00** [comment]
> Lagspike when going in and out of water with 10 planets
> 
> image

done

**Review by isakswe — APPROVED** [review]
Looks good to me 👍

File: Assets/Scripts/Foliage/SpawnFoliage.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 public class SpawnFoliage : MonoBehaviour
 {
-    const int prefabLimit = 20000;
```
Comment (isakswe) [inline Assets/Scripts/Foliage/SpawnFoliage.cs:14]: Only thing that's bugging me. Why is this changed? Andy why in this PR?

Comment (Hansson00) [reply Assets/Scripts/Foliage/SpawnFoliage.cs]: Fixed, must have been something I forgot when testing

**Comment by Danilll01** [comment]
Looks good!
